web/templates/cluster_hana.html.tmpl

```diff
@@ -102,17 +98,15 @@
     <h3>Pacemaker Site details</h3>
     <div class="row mt-4">
         <div class="col-xl-12">
-            {{ template "sites" .Nodes.GroupBySite }}
+            {{ template "sites" .Cluster.Details.Nodes.GroupBySite }}
         </div>
     </div>
     <hr>
 
     <h3>SBD/Stonith</h3>
-    {{- if .Cluster.IsFencingSBD }}
-        {{ template "sbd" .Cluster.SBD }}
-    {{- end }}
+    {{ template "sbd" .Cluster.Details.SBDDevices }}
```

Comment:
I know that by now we always use SBD as fencing, but having the native Azure fencing system is quite normal in production systems. With this change, we would have an empty div after this `h3` header with empty values.
Not a big deal, but I just wanted to mention it

Reply:
I will hide  the h3 + block if no SBDDevice is present